.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
@flapflapio/core, not sure if we want to exclude these IDE-related files or not. Are you guys using pycharm or vscode?

Reply:
We should exclude them. I am going to use pycharm for the selenium tests. Ill ask amneet what he will use